src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTypeCheck.java

```diff
@@ -94,7 +97,7 @@
  * </li>
  * <li>
  * Property {@code allowedAnnotations} - Specify annotations that allow
- * missed documentation. Only short names are allowed, e.g. {@code Generated}.
+ * skipping validation at all. Only short names are allowed, e.g. {@code Generated}.
```

Comment:
What was wrong with original wording?

Reply:
"Documentation" word is very confusing.

Reply:
Here is the original discussion

https://github.com/checkstyle/checkstyle/pull/12736#issuecomment-1427245513